js/calculators/assessmentResultCalculator.js

```diff
@@ -0,0 +1,121 @@
+var inRange = require( "lodash/number/inRange" );
+var isUndefined = require( "lodash/lang/isUndefined" );
+var replaceAll = require( "../helpers/replaceAll.js" );
+
+/**
+ * Check whether the current item can be seen as a range.
+ * @param {object} entry The entry in the assessment configuration.
+ * @returns {boolean} Whether or not a range is available.
+ */
+var hasRange = function( entry ) {
+	return entry.hasOwnProperty( "range" ) || ( entry.hasOwnProperty( "min" ) && entry.hasOwnProperty( "max" ) );
+};
+
+/**
+ * Check whether the result falls within a certain range.
+ * @param {object} entry The entry in the assessment configuration.
+ * @param {number} result The result given by the analysis.
+ * @returns {boolean} Whether or not the entry falls within the given range.
+ */
+var entryInRange = function( entry, result ) {
+	if ( entry.hasOwnProperty( "range" ) ) {
+		return inRange( result, entry.range[0], entry.range[1] );
+	}
+
+	return inRange( result, entry.min, entry.max );
+};
+
+/**
+ * Check whether only a min property is available.
+ * @param {object} entry The entry in the assessment configuration.
+ * @returns {boolean} Whether or not the entry has a min property and no max property.
+ */
+var hasMinimum = function( entry ) {
+	return ( entry.hasOwnProperty( "min" ) && !entry.hasOwnProperty( "max" ) );
```

Comment:
Maybe we should rename this to hasOnlyMinimum, since a scoring can both have a max and a min.